test/system/070-build.bats

```diff
@@ -31,4 +31,23 @@ EOF
     run_podman rmi build_test
 }
 
+@test "podman build - test RUN with priv'd command" {
+    if [[ "$PODMAN" =~ -remote ]]; then
+        if [ "$(id -u)" -ne 0 ]; then
+            skip "unreliable with podman-remote and rootless; #2972"
+        fi
+    fi
+
+    tmpdir=$PODMAN_TMPDIR/build-test
+    run mkdir -p $tmpdir || die "Could not mkdir $tmpdir"
+    dockerfile=$tmpdir/Dockerfile
+    cat >$dockerfile <<EOF
+FROM alpine 
+RUN apk add nginx
+EOF
+
+    run_podman build -t build_test $tmpdir
+    is "$output" ".*STEP 3: COMMIT" "COMMIT seen in log"
+    run_podman rmi -a -f 
```

Comment:
please just `rmi build_test`, not `-a`. This will let subsequent tests use the cached $IMAGE.

Reply:
fixed